File: EXERCÍCIOS PARA ENTREGA/Exercício 14.c
//Entrar via teclado com três valores distintos. Exibir o maior deles.
#include <stdio.h>

int main(int argc, char *argv[])
{
 float n1, n2, n3, maior;

    printf("Digite os numeros para a comparacao:\n");
    scanf("%f \n%f \n%f", &n1, &n2, &n3);

    maior = n1;
    if(n2 > maior)
        maior = n2;
    if(n3 > maior)
        maior = n3;

    printf("\nO maior numero  informado e: %.2f", maior);

    return 0;
}

File: PYTHON/Exercício 14.py
# Entrar via teclado com três valores distintos. Exibir o maior deles.

n1 = float(input('Digite o primeiro número: '))
n2 = float(input('Digite o segundo número: '))
n3 = float(input('Digite o terceiro número: '))

if n1>n2 and n1>n3:
 print(f"O número maior é:{n1:.2f}")
elif n2>n1 and n2>n3:
 print(f"O número maior é:{n2:.2f}")
else:
 print(f"O número maior é: {n3:.2f}")